cpp/ql/test/experimental/query-tests/Security/CWE/CWE-703/semmle/tests/test.cpp

```diff
@@ -0,0 +1,26 @@
+typedef unsigned int size_t;
+void clean();
+
+
+void funcTest1()
+{
+   throw ("my exception!",546);  // BAD
```

Comment:
Can you explain what's going on here?

Reply:
It seems to me that such a construction may appear when the name of the function is lost in front of the brackets. it will compile and work. but functionally useless.

Reply:
I see, thanks.